tests/test_ssl_transport_close.py

```diff
@@ -0,0 +1,144 @@
+import asyncio
+import warnings
+from typing import Any, Tuple, cast
+
+import pytest
+
+from aiohttp import TCPConnector, web
+from aiohttp.client import ServerDisconnectedError
+
+TestServer = Any
+TestClient = Any
+
+
+async def handle_root(request: web.Request) -> web.Response:
+    """respond after server was restarted, forcing connection_lost"""
+    cq = request.app["cq"]
+    dq = request.app["dq"]
+
+    await dq.put(0)
+
+    await cq.get()
+    cq.task_done()
+
+    return web.Response(text="")
+
+
+async def _prepare(
+    aiohttp_server: Any,
+    aiohttp_client: Any,
+    ssl_ctx: Any,
+    client_ssl_ctx: Any,
+    cq: asyncio.Queue[int],
+    dq: asyncio.Queue[int],
+) -> Tuple[TestServer, TestClient]:
```

Comment:
I know many of our tests are still labelled with `Any`, but I'm in the process of cleaning it up. Could you add in the correct annotations here? Let me know if there's anything you can't find the type of.

Off the top of my head, TestClient/TestServer are in aiohttp.test_utils, ssl_ctx is `ssl.SSLContext`, and any other fixtures can be checked in the source code (they should have the return type annotated correctly).

Reply:
try regexxer for the cleanup.

Current
```
AiohttpClient = Callable[[Union[Application, BaseTestServer]], Awaitable[TestClient]]
AiohttpServer = Callable[[Application], Awaitable[TestServer]]
```
do not have the kwargs parameters for port & ssl_ctx, which is why I have to declare them as Any in one location.